documentation/source/user_guide_model_creation.rst

```diff
@@ -529,4 +529,240 @@ The following image shows the simulation results:
+    - **I**: Current through the winding, with similar representation to voltage.
+
+  - **core_eddy_losses** and **core_hyst_losses**: These values represent the losses due to eddy currents and hysteresis in the core,
+    crucial for evaluating core material performance under alternating magnetic fields.
+- **total_losses**: This section summarizes the overall energy dissipation within the system, combining losses from various components.
+  It is broken down into several key areas:
+
+    - **winding1**: It indicates the total losses within every winding, which is the sum of power losses across all turns in this winding.
+
+    - **all_windings**: It aggregates the losses across all windings in the system.
+
+    - **core**: This repeats the total losses within the core.
+
+    - **total_losses**: it represents the sum of all losses within the system, including windings and core.
+
+log_electro_magnetic.json File in Time Domain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is an example of how the outcomes of time domain simulation are structured.
+
+.. code-block:: json
+
+    {
+      "time_domain_simulation": [
+        {
+          "f": 100000,
+          "T": 1e-05,
+          "Timemax": 1.25e-05,
+          "number_of_steps": 5,
+          "dt": 2.5e-06
+        },
+        {
+          "step_1": {
+            "windings": {
+              "winding1": {
+                "number_turns": 10,
+                "flux": [-7.209142581890734e-06],
+                "V": [-2.892335944263035],
+                "I": 2.0
+              },
+              "winding2": {
+                ... }}}
+        },
+        {
+         "...": {}
+      ],
+      "average_losses": {
+        "core_eddy_losses": [0.00037330739608363716],
+        "core_hyst_losses": [0],
+        "winding1": {
+          "winding_losses": 1.2578033060966147,
+          "flux_over_current": [6.703043813058433e-06],
+          "average_current": [0.4],
+          "average_voltage": [1.8009901071754213],
+          "P": 0.7203960428701686,
+          "S": 4.5274565545301515,
+          "Q": 4.469775429993662
+        },
+        "winding2": {...}
+      },
+      "total_losses": {
+        "all_windings_losses": 2.511429275334878,
+        "eddy_core": 0.00037330739608363716,
+        "core": 0.00037330739608363716,
+        "total_losses": 2.5118025827309616
+      }
+    }
+
+**Key Components Explained:**
+
+- **time_domain_simulation**: An array capturing simulation steps over time, including initial setup and individual time steps.
+
+  - The initial setup specifies the simulation frequency (**f**), period (**T**), maximum time (**Timemax**), total number of steps
+    (**number_of_steps**), and time increment (**dt**).
+
+  - **step_1** and subsequent steps detail the state of windings at specific times. For example, in **step_1**, **winding1** shows:
+
+    - **number_turns**: Number of turns in the winding.
+    - **flux**: Magnetic flux through the winding.
```

Comment:
do not forget real and imaginary part of flux, V, I

Reply:
we do not have it in time domain result

Reply:
The flux over current in time domain may be not calculated correctly. I discussed this before with Till and we look to it in the future

Reply:
May leave a hint that is it not correct calculated for time-domain simulations